datasource/threedi_results.py

```diff
@@ -219,8 +128,29 @@ def get_timestamps(self, parameter=None):
                 parameter
             )
 
+    def get_gridadmin(self, variable=None):
+        """Return the gridadmin where the variable is stored. If no variable is
+        given, a gridadmin without results is returned.
+
+        Results are either stored in the 'results_3di.nc' or the
+        'aggregate_results_3di.nc'. These make use of the GridH5ResultAdmin and
+        GridH5AggregateResultAdmin to query the data respectively.
+
+        :param variable: str of the variable name, e.g. 's1', 'q_pump'
+        :return: handle to GridAdminResult or AggregateGridAdminResult
+        """
+        if variable is None:
+            return self.gridadmin
+        elif variable in self.available_subgrid_map_vars:
+            return self.gridadmin_result
```

Comment:
"gridadmin_result" is "het resultaat van de gridadmin".
"result_gridadmin" maakt het duidelijker dat het een gridadmin is?
Ik weet niet, ik zit nog met de namen te spelen.

`regular_gridadmin` en `aggregate_gridadmin`?

Of is het meer "resultaat" dan "gridadmin"? 

`regular_result` en `aggregate_result`?

Reply:
Als we kijken naar `threedigrid` worden de volgende klassen gebruikt:
1. `GridH5Admin`
2. `GridH5ResultAdmin`
3. `GridH5AggregateResultAdmin`

De volgende namen zijn dan misschien wel consequent:
1. `grid_h5_admin`
2. `grid_h5_result_admin`
3. `grid_h5_aggregate_result_admin`

Maar de namen worden toch wel erg lang, iets korter:
1. `gridadmin` (alleen `admin` kan misleidend zijn)
2. `result_admin`
3. `aggregate_result_admin`